docs/guides/misc/autoretirement.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Auto-retirement for Inactive Users
+---
+
+Documentation for hackmud's automatic inactive account retirement.
+
+## Accounts with no activity for a duration of 60 days will automatically retire all users
+
+In order to preserve performance of the Multi-User Domain over time, it is necessary to remove users and records from the database. Users that belong to accounts which have gone inactive for 60 days will automatically retire.
+
+## Email Reminders
+
+Players who wish to be notified of retirement can sign up for reminder emails by creating a [hackmud forums](https://hackmud.com/forums) account, linked to the steam account which owns hackmud.
```

Comment:
there is no 'forums account' -- it's the same account.

The instructions are 'log into the forums with the same steam account which owns hackmud', then go to your account page (hackmud.com/accounts/edit) and add an email and make sure the checkbox is checked for retire notifications.